site/src/pages/workshop/workshop.md

```diff
@@ -375,9 +375,9 @@ export const App: BlockComponent<BlockEntityProperties> = ({
 }) => {
   ...
   // find only the entities in linkedEntities which appear as the destination of one of the projectLinks
-  const 
-  
-  
+  const
```

Comment:
This file has been fixed in https://github.com/blockprotocol/blockprotocol/pull/641. I'm going to merge `main` and accept remote changes to make this PR ready to go 🙂